bin/_main

```diff
@@ -278,10 +278,19 @@ if [ "${1}" == "log" ]; then
     shift
     crucible_log ${1} ${LOG_DB}
     EXIT_VAL=$?
-elif [ "${1}" == "ls" -o "${1}" == "tags" ]; then
+elif [ "${1}" == "ls" -o "${1}" == "tags" -o "${1}" == "result-completion" ]; then
```

Comment:
result-completion ? isn't just "completion" ?

Reply:
It comes down to a context issue.  I have wondered if I chose the right way, but for the moment I have decided to stay with it.  I'm curious if you agree when I explain my reasoning.

In the scope of crucible, it is helpful to denote the crucible command (`crucible result-completion`) as specific to results in case we expand our tab completion querying in the future for things other than results (using some other script(s)).  Image something like `crucible benchmark-completion` as a future improvement (if you look at the `_crucible_completions` file you will see we are doing that with some find magic today).

For the script that generates the tab completion data (in this case `result-processor.py`) it somewhat redundant to label it `result-completion` when all it handles is results.

So ultimately, when you ask crucible for help with result-completion it knows to ask the result processor script for its completion output.

Thoughts?

Reply:
Thanks for explaining. Sounds reasonable.